### PR TITLE
Add TrafficSelectorPolicy param in VpnConnection for VWAN

### PR DESCRIPTION
**Pull request by akshatkale**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/Network/Network/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/862
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by akshatkale** [comment]
Waiting for - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/862

**Comment by akshatkale** [comment]
Waiting on Cmdlet review to finish - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/862

File: src/Network/Network/ChangeLog.md
Comment (akshatkale) [inline src/Network/Network/ChangeLog.md:31]: These Reset changes were added in PR https://github.com/Azure/azure-powershell/pull/14225 but ChangeLog in that PR was incorrect so fixed it.

**Review by wyunchi-ms — APPROVED** [review]

**Comment by wyunchi-ms** [comment]
LGTM

**Comment by akshatkale** [comment]
@wyunchi-ms - I am not authorized to merge PRs in this repo. I think you'll have to merge it in.

**Comment by wyunchi-ms** [comment]
Please resolve the conflicts.

**Comment by akshatkale** [comment]
@wyunchi-ms Resolved. Please merge today once the checks pass. Also you might want to close the Cmdlet review https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/862 before merging this PR?

**Pull request merged**
